_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -35,6 +35,19 @@
 #define MFX_CHECK_WITH_ASSERT(EXPR, ERR) { assert(EXPR); MFX_CHECK(EXPR, ERR); }
 //#define PARALLEL_BRC_support
 
+inline mfxStatus CheckAndDestroyVAbuffer(VADisplay display, VABufferID & buffer_id)
+{
```

Comment:
What the reason to introduce one more kind of MFX_DESTROY_VABUFFER?

Reply:
Because macro MFX_DESTROY_VABUFFER doesn't check returned result.